.php_cs.dist

```diff
@@ -0,0 +1,50 @@
+<?php
```

Comment:
No, I'm asking whether `php-cs-fixer` checks any other location besides current folder by default. If not - no need to move configuration file.